app/scripts/lib/transaction/util.test.ts

```diff
@@ -182,6 +201,78 @@ describe('Transaction Utils', () => {
 
         expect(completed).toBe(true);
       });
+
```

Comment:
nice tests overall

suggestion: we could wrap the tests when ppom is enabled with
```
describe('when blockaid is enabled', () => {
  beforeEach(async () => {
        await addTransaction({
          ...request,
          securityAlertsEnabled: true,
          chainId: '0x1',
        });  
  });
  
  it('validates if the chain is supported', () => {
  });
  
  it('does not validate if the chain is not supported', () => {
  });
  }
}
```
- this also updates 'validates if ppom is enabled' → 'validates if the chain is supported'
- this also updates 'ppom' → 'blockaid'

sidenote: I have a bit of an outstanding  comment with blockaid vs ppom. ppom seems to be a part of Blockaid and can be used as a general term for other projects

Reply:
that's correct. the code above only is an example for only when blockaid is enabled. For when blockaid is not enabled, we can keep the same test standalone or we could also add it into a another describe block

```
describe('when blockaid is enabled', () => {
  beforeEach(async () => {
        await addTransaction({
          ...request,
          securityAlertsEnabled: true,
          chainId: '0x1',
        });  
  });
  
  it('validates if the chain is supported', () => {
  });
  
  it('does not validate if the chain is not supported', () => {
  });
  }
}

it('does not validate', () => {
});
```
// or
```

describe('when blockaid is enabled', () => {
  // beforeEach hook setting each test
  // tests
});

describe('when blockaid is not enabled', () => {
  it('does not validate', () => {
  });
});
```